From cfea911c6c57eebdfc1358ee47b3730514da6cf3 Mon Sep 17 00:00:00 2001
From: \\\"Mike (mwester)\\ <mwester@dls.net>
Date: Wed, 19 Nov 2008 13:11:56 +0000
Subject: [PATCH] change-gta01-battery-driver-name.patch

[Can this be added to "stable" please?  Thanks!  Simple, short,
non-descriptive names seem to be the norm for the nodes beneath
/sys/classes/power_supply, so there seems to be no reason to do
differently for the GTA01, especially when doing so adds an
unnecessary incompatibility.]

    change-gta01-battery-driver-name.patch

    This patch changes the name of the gta01 battery driver to "bat"
    in order to maintain compatability with the gta02 device.

    Signed-off-by: Mike Westerhof <mwester@dls.net>
---
 drivers/power/gta01_battery.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/power/gta01_battery.c b/drivers/power/gta01_battery.c
index 5acb45c..2267ecf 100644
--- a/drivers/power/gta01_battery.c
+++ b/drivers/power/gta01_battery.c
@@ -70,7 +70,7 @@ static int __init gta01_bat_init(void)
 {
 	int ret;
 
-	bat_pdev = platform_device_register_simple("gta01-battery", 0, NULL, 0);
+	bat_pdev = platform_device_register_simple("bat", 0, NULL, 0);
 	if (IS_ERR(bat_pdev))
 		return PTR_ERR(bat_pdev);
 
-- 
1.5.6.5

